dashboard/app/helpers/levels_helper.rb

```diff
@@ -107,6 +107,16 @@ def get_channel_for(level, user = nil)
     channel_token&.channel
   end
 
+  # If given a level and a user and optional script, returns whether the level
+  # has been started by the user
+  def level_started?(level, user, script = nil)
+    if level.channel_backed? # maureen make sure user is present or this will fail
```

Comment:
So...  have you made sure the user is present?  :-)

Reply:
This is why I leave notes for myself! Thanks for catching that, will update.